docs/decoupled_models.md

```diff
@@ -51,6 +53,20 @@ real deployment, the backend should not allow the caller thread to return from
 TRITONBACKEND_ModelInstanceExecute until that instance is ready to handle another
 set of requests.
 
+### Python model using Python Backend \[BETA\]
+
+Read carefully about the [Python Backend](https://github.com/triton-inference-server/python_backend),
+and specifically [`execute`](https://github.com/triton-inference-server/python_backend#execute).
+
+The [decoupled examples](https://github.com/triton-inference-server/python_backend/tree/main/examples/decoupled)
+demonstrates how decoupled API can be used to implement a decoupled
+python model. However, the example is designed to show the flexibility of the
+decoupled API and in no way should be used in production. This example circumvents
+the restriction placed by the [instance count](model_configuration.md#instance-groups)
```

Comment:
I think we already mention this in the Python backend docs so perhaps we can remove it from here?

Reply:
@tanmayv25 , @Tabrizian  could you please wrap it up

Reply:
@tanmayv25 , @Tabrizian  could you please wrap it up